src/tabs/firmware_flasher.html

```diff
@@ -3,6 +3,16 @@
         <div class="options gui_box">
             <div class="spacer">
                 <table class="cf_table" style="margin-top: 10px;">
+                    <tr>
+                        <td>
+                            <select name="build_type">
+                                <option value="release">Release</option>
+                                <option value="release_candidate">Release Candidate</option>
```

Comment:
Not sure if these (and the other entries for this list) should be translated? @McGiverGim?

Reply:
Yes, we translate it too.